### PR TITLE
[CosmosDB] Adds support to create containers with Client Encryption Policy

### PR DESCRIPTION
**Pull request by kr-santosh**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by abhijitpai — DISMISSED** [review]

File: src/CosmosDB/CosmosDB/help/New-AzCosmosDBSqlContainer.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ New-AzCosmosDBSqlContainer -ResourceGroupName <String> -AccountName <String> -Da
  [-AutoscaleMaxThroughput <Int32>] [-TtlInSeconds <Int32>] [-UniqueKeyPolicy <PSSqlUniqueKeyPolicy>]
  [-ConflictResolutionPolicyMode <String>] [-ConflictResolutionPolicyPath <String>]
  [-ConflictResolutionPolicyProcedure <String>] [-ConflictResolutionPolicy <PSSqlConflictResolutionPolicy>]
- [-AnalyticalStorageTtl <Int32>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
- [<CommonParameters>]
+ [-ClientEncryptionPolicy <PSSqlClientEncryptionPolicy>] [-AnalyticalStorageTtl <Int32>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```
Comment (BethanyZhou) [inline src/CosmosDB/CosmosDB/help/New-AzCosmosDBSqlContainer.md:24]: Could you please provide one example about how to create PSSqlClientEncryptionPolicy object for customer?

Comment (kr-santosh) [reply src/CosmosDB/CosmosDB/help/New-AzCosmosDBSqlContainer.md:24]: the reason why i did not add is there is just an example on how to create a customer. But no example for the policies / optional parameters we support.

Comment (BethanyZhou) [reply src/CosmosDB/CosmosDB/help/New-AzCosmosDBSqlContainer.md:24]: It is nice to have an example to demonstrate the usage of complex parameters. We have had many cases that customer created a GitHub issue to ask how to use a complex parameter. But no worries, the example is not mandatory.

Comment (kr-santosh) [reply src/CosmosDB/CosmosDB/help/New-AzCosmosDBSqlContainer.md:24]: Yup. Looks like i need to re-record all the tests. Will update the PR once all the tests run.

Comment (kr-santosh) [reply src/CosmosDB/CosmosDB/help/New-AzCosmosDBSqlContainer.md:24]: fixed.

**Review by BethanyZhou — APPROVED** [review]
LGTM.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**